contrib/comdb2makecluster

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env bash
```

Comment:
Can you please place the script under `contrib/comdb2makecluster/`, but not directly under `contrib/`?

Reply:
Will do.